features/buyers/accounts/edit.feature

```diff
@@ -19,12 +19,6 @@ Feature: Audience > Accounts > Edit
     Then the current page is the buyer account page for "Pepito"
     And should see the flash message "Account successfully updated"
 
-  Scenario: Required fields and validation
-    Given they go to the buyer account edit page for "Pepe"
-    When the form is submitted with:
-      | Organization/Group Name | |
-    Then field "Organization/Group Name" has inline error "can't be blank"
```

Comment:
you mean that this test breaks with the HTML required?

Reply:
Yes, because you just cannot submit a form with the required field being empty.

Reply:
Looks good. I am wondering though, can any of the answers here be used to validate that these fields do have the `required` property?

https://stackoverflow.com/questions/6114385

Or maybe we have some integration test where we can check the html of the form?

Reply:
> Or maybe we have some integration test where we can check the html of the form?

I don't think we have such a thing... but maybe I'm wrong.

Not sure what we need to add - whether these checks from SO (but I guess it is browser-dependant), or maybe unit tests for `output_html` method of the patternfly form builder.

But I'd vote for merging this PR as it is, and improve later :upside_down_face: